index.html

```diff
@@ -30,73 +30,276 @@ <h2 class="logo">
-          A daily selection of privately<br>
-          personalized reads; no accounts or<br>
-          sign-ups required.</p>
-
-        <ul class="skills">
-          <li>html</li>
-          <li>css</li>
-          <li>javascript</li>
-        </ul>
-        <button type="button" class="linktonic">See Project</button>
       </article>
 
+      <article>
+        <div class="container">
+          <div class="works-tonic">
+            <div class="snapshoot">
+              <img src="./images/Snapshoot-1.png" alt="tonich4" />
+            </div>
+            <div class="work-title">
+              <h4>Tonic</h4>
+              <ul class="works-info">
+                <li class="works-ref-font">CANOPY</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>Back End Dev</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>2015</li>
+              </ul>
+              <p class="text-tonic">
+                A daily selection of privately<br>
+                personalized reads; no accounts or<br>
+                sign-ups required.</p>
+              <ul class="skills">
+                <li class="li-html">html</li>
+                <li class="li-css">css</li>
+                <li class="li-javascript">javascript</li>
+              </ul>
+            </div>
+
+            <div class="link-project"></div>
+            <a class="linktonic" href="#">See Project</a>
+          </div>
+        </div>
+      </article>
+
+      <article>
+        <div class="container">
+          <div class="works-tonic">
+            <div class="snapshoot">
+              <img src="./images/Snapshoot-1.png" alt="tonich4" />
+            </div>
+            <div class="work-title">
+              <h4>Tonic</h4>
+              <ul class="works-info">
+                <li class="works-ref-font">CANOPY</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>Back End Dev</li>
+                <li><span class="counter"><img src="./images/counter.png" alt="counter" /></span></li>
+                <li>2015</li>
+              </ul>
+              <p class="text-tonic">
+                A daily selection of privately<br>
+                personalized reads; no accounts or<br>
+                sign-ups required.</p>
+              <ul class="skills">
+                <li class="li-html">html</li>
+                <li class="li-css">css</li>
+                <li class="li-javascript">javascript</li>
+              </ul>
+            </div>
+
+            <div class="link-project"></div>
+            <a class="linktonic" href="#">See Project</a>
+          </div>
+        </div>
+      </article>
+
+    </section>
+
+    <section class="about">
+      <div class="container">
+        <div class="back-images-intro">
+
+          <h1>About<br>Myself</h1>
```

Comment:
- [ ] Kindly fix the design 
- Your Design
![Captureabu](https://user-images.githubusercontent.com/81444338/171225199-e4821859-a898-4a9d-8a5b-aa323b1942b5.PNG)
![Capturemys](https://user-images.githubusercontent.com/81444338/171225307-644efb01-9906-4f2a-9045-3cb19cde2c71.PNG)
- Figma Design
![Captureabfig](https://user-images.githubusercontent.com/81444338/171225422-ba811970-6764-4535-b690-bb3b7067ee0f.PNG)